src/hooks/data/useMgnifyCobsSearch/index.tsx

```diff
@@ -27,7 +46,7 @@ const useMgnifyBigsiSearch: (
       body: formData,
     }
   );
-  return data as MGnifyResponseGenericObj;
+  return data as unknown as CobsResponse;
```

Comment:
Why the "unknown" here?

Reply:
The `useData` hook is one we use in lots of places. I could have added the `CobsResponse` to the set of allowable types in the return type of `useData`... but since we only use it in this one special place, I thought it was cleaner to keep that type local. Because it doesn't overlap with any of the other allowable types, you have to explicitly cast via `unknown` to convert between two non-obviously-overlapping types.

Reply:
ah, cool. Thanks!